build/pmm-client-docker/entrypoint.py

```diff
@@ -1,3 +1,50 @@
 #!/usr/bin/env python
+'''
+Basic Kubernetes-compatible PMM Client Docker entrypoint.
 
-print("PMM CLIENT 2.X DOCKER")
+To configure pmm-agent set environment variable PMM_AGENT_SETUP=true.
+This will run:
+`pmm-agent setup --config-file=/usr/local/percona/pmm2/config/pmm-agent.yaml`
+'''
+
+from __future__ import print_function, unicode_literals
+from distutils.util import strtobool
+from os import environ, execlp, EX_OK
+from subprocess import Popen
+import sys
+
+
+# if PMM_AGENT_SETUP is True run:
+# `pmm-agent setup --config-file=/usr/local/percona/pmm2/config/pmm-agent.yaml`
+PMM_AGENT_SETUP = False
+
+PMM_AGENT_CONFIG = '/usr/local/percona/pmm2/config/pmm-agent.yaml'
+
+
+def pmm_agent_setup():
+    '''Setup pmm-agent.'''
+    cmd = ('pmm-agent', 'setup', '--config-file=%s' % (PMM_AGENT_CONFIG,))
+    proc = Popen(cmd, universal_newlines=True)
+    return proc.wait()
+
+
+def pmm_agent_run():
+    '''Run pmm-agent.'''
+    execlp('pmm-agent', '--config-file=%s' % (PMM_AGENT_CONFIG,))
```

Comment:
@askomorokhov you wanted to remove those small functions?

Reply:
Done.